CHANGELOG.en-US.md

```diff
@@ -0,0 +1,119 @@
+---
+order: 3
+title: Change Log
+toc: false
+timeline: true
+---
+
+If you want to read change logs before `2.0.0`, please visit [GitHub](https://github.com/ant-design/ant-design/releases?after=2.0.0).
+
+---
+
+很高兴的通知各位，经过四个月时间的紧密开发，`antd@2.0.0` 终于发布了。这个版本我们重构了底层代码，持续完善现有组件功能和优化细节，其中很多都来自社区的贡献，无法一一感谢，欢迎各位持续关注和鞭策。在升级过程中遇到任何问题，请及时反馈给我们。
+
+### 2.x Major changes
+
+* Refactor components with TypeScript, and provide **`.d.ts` files which are supported officially**. Thank all the developers in [#1846](https://github.com/ant-design/ant-design/issues/1846) and @infeng.
+* **Translate the documentation into English**, and we are going to provide both of Chinese and English documentation in the future. Thank all the translators and reviewers in [#1471](https://github.com/ant-design/ant-design/issues/1471).
+* DatePicker, TimePicker, Calendar and other components that are designed to select time **are refactored to replace [gregorian-calendar](github.com/yiminghe/gregorian-calendar) with [moment](http://momentjs.com/)**.
+* All the [icons](http://ant.design/components/icon/) are re-designed.
+* New component [Mention](http://ant.design/components/mention/).
+* New component [AutoComplete](http://ant.design/components/auto-complete/).
+* The `getFieldProps` of Form is replaced with `getFieldDecorator` which will warn developers if they make mistakes. Related discussion [#1533](https://github.com/ant-design/ant-design/issues/1533).
+* Table supports [grouping columns](http://ant.design/components/table/#components-table-demo-grouping-columns). @yesmeck
+* Removed components and features which are deprecated in `antd@1.x`, such as QueueAnim, Validation, Form.ValueMixin, Progress.Line, Progress.Circle, Popover[overlay] and Slider[marks] will not support array any more.
+
+### 2.x Breaking changes
+
+There are some breaking changes in `antd@2.0.0`, and you need to modify your code to work with it.
+
+* `value` and `defaultValue` of all the time-related components will not support type `String/Date`, please use [moment](http://momentjs.com/):
+  ```diff
+  - <TimePicker defaultValue="12:08:23" />
+  + <TimePicker defaultValue={moment('12:08:23', 'HH:mm:ss')} />
+
+  - <DatePicker defaultValue="2015/01/01" />
+  + <DatePicker defaultValue={moment('2015/01/01', 'YYYY/MM/DD')} />
+
+  - <Calendar defaultValue={new Date('2010-10-10')} />
+  + <Calendar defaultValue={moment('2010-10-10', 'YYYY-MM-DD')} />
+  ```
+* Parameters that are type `Date/GregorianCalendar` of `onChange` and `onPanelChange` and other callback function had been change to type moment. Please consult [APIs of gregorian-calendar](https://github.com/yiminghe/gregorian-calendar) and [APIs of moment](http://momentjs.com/docs/), and update you code. Because the reutrn value of `JSON.stringy(date: moment)` will lost time zone, we should use `.format` to convert date to string first, related issue [#3082](https://github.com/ant-design/ant-design/issues/3082):
```

Comment:
* Parameters of type `Date/GregorianCalendar` of functions such as `onChange` and `onPanelChange`, plus other callback functions had been changed to type moment. Please consult [APIs of gregorian-calendar](https://github.com/yiminghe/gregorian-calendar) and [APIs of moment](http://momentjs.com/docs/), and update your code accordingly. Because the return value of `JSON.stringy(date: moment)` will lost time zone, we should use `.format` to convert date to string first, see related issue [#3082](https://github.com/ant-design/ant-design/issues/3082) for details: